### ISSUE TITLE
ExtraMovementState not changing with Server Authoritative Movement

### ISSUE THREAD

**Issue by ikay70** [suriyun-production/mmorpg-kit-docs#216]
Hey!
Trying to figure out what's happening with ExtraMovementState under Enttiy Movement Settings set to ServerAuthoritative (SyncMode: ServerToClients). Is it not working by design or is it just me?

Im trying it with RigidBodyEntityMovement and almost standard PlayerCharacterController.

And when I change MovementSecure to NotSecure, it does change on local client - character do sprint -, but on other clients the character is simply not moving.

Thanks!

update:
if I do comment following condition in BaseGameEntity.cs::public void SetExtraMovement, everthing starts to work:
  
```
if (MovementSecure == MovementSecure.ServerAuthoritative && IsServer)               
         ExtraMovementState = extraMovementState;

     //if (MovementSecure == MovementSecure.NotSecure && IsOwnerClient)                
       CallNetFunction(NetFuncSetExtraMovement, DeliveryMethod.Sequenced, FunctionReceivers.Server, (byte)extraMovementState);
```

**Comment by insthync** [comment]
This is a bug, I have to fix it, thank you for the report and tell me how you fix it, I will try :)

**Comment by insthync** [comment]
This was fixed in 1.53. If this issue still occurs, you can reopen this issue post.
